lib/crypto/tasks.py

```diff
@@ -264,6 +268,11 @@ def file_supports_firefox(version):
         to_unsign = version.files.filter(ff_version_filter,
```

Comment:
I don't think so: the idea is that we first get all the versions for addons that are either extensions or themes, because those are the only kind of addons that can have multi-package XPIs.

In the `if` below we then make sure that we're only keeping the files that are really multi-package if the add-on is a theme.

Does that make sense?